tests/framework/CodeInterfaceTests/test_generic_interface.xml

```diff
@@ -27,6 +27,7 @@
     <Input name="one.xml" type="">one.xml</Input>
     <Input name="inp.two" type="">inp.two</Input>
     <Input name="mesh" type="">dummy.e</Input>
+    <Input name="a_dummy_file_for_subdirectory" type="" subDirectory="testSubDirectory">dummy_file_for_subdirectory.dummy</Input>
```

Comment:
This is a UserGeneated input file. The attribute just specifies that a subDirectory needs to be generated in the sampling process. For example,
In a MultiRun, we will copy the file ```dummy_file_for_subdirectory.dummy``` from ```workingDir/subDirectory/``` to the location ```workingDir/stepName/1/subDirectory/```